vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.h

```diff
@@ -41,16 +41,21 @@ class ReactContext final : public Mso::UnknownObject<IReactContext> {
  public:
   ReactContext(
       Mso::WeakPtr<ReactInstanceWin> &&reactInstance,
-      winrt::Microsoft::ReactNative::IReactPropertyBag const &properties) noexcept;
+      winrt::Microsoft::ReactNative::IReactPropertyBag const &properties,
+      winrt::Microsoft::ReactNative::IReactNotificationService const &notifications) noexcept;
+
+  void Destroy() noexcept;
```

Comment:
>void [](start = 2, length = 4)

Can you add quick comment on why should users need this as opposed to just having destructor taking care of this?